project/plugins.sbt

```diff
@@ -19,3 +19,4 @@ addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.25")
 addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.8")
 addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
+addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.2.4")
```

Comment:
Latest is 0.2.6 (looks like we forgot to update the example projects)

Reply:
Indeed... my proof that I _did_ Read The Fantastic Manual, but missed the important bit.